wgs/workflows/postprocessing/genome_wide_plot.py

```diff
@@ -0,0 +1,171 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+import matplotlib.backends.backend_pdf
+
+from wgs_qc_utils.plotter import variant_plotting, coverage_plotting, \
```

Comment:
and we're going to have to install this in the main container now. 

so can you please also add that to the wgs container. and this repo will require proper branches and version releases from now on.

Reply:
yes